src/Fixer/LanguageConstruct/ClassKeywordFixer.php

```diff
@@ -44,6 +44,16 @@ public function getDefinition(): FixerDefinitionInterface
         );
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * Must run after FullyQualifiedStrictTypesFixer.
```

Comment:
It looks like you've added this by hand, but it's not valid. You should add:

```
'class_keyword' => [
                'fully_qualified_strict_types',
            ],
```

to `PhpCsFixer\Tests\AutoReview\FixerFactoryTest::getFixersPriorityGraph()` and then when you run this test you'll get:

```
There are 2 priority PHPDoc issues found.

--------------------------------------------------
[class_keyword] PHPDoc for ClassKeywordFixer::getPriority is missing.
Expected:
/**
     * {@inheritdoc}
     *
     * Must run before FullyQualifiedStrictTypesFixer.
     */
--------------------------------------------------
[fully_qualified_strict_types] PHPDoc for FullyQualifiedStrictTypesFixer::getPriority is not as expected.
Expected:
/**
     * {@inheritdoc}
     *
     * Must run before NoSuperfluousPhpdocTagsFixer, OrderedImportsFixer, OrderedInterfacesFixer, StatementIndentationFixer.
     * Must run after ClassKeywordFixer, PhpdocToReturnTypeFixer.
     */
```

There will be also fail in `testFixersPriorityCasesHaveIntegrationTest`, you need to add proper integration test. Please follow `composer qa` and make it pass 🙂.